containers/unit-tests/start

```diff
@@ -86,7 +86,7 @@ class SourceDirectory(PodmanTemporaryDirectory):
             if args.wait:
                 run([f'{self.name}/tools/webpack-jumpstart', '--wait'], check=True)
             elif not args.no_jumpstart:
-                run([f'{self.name}/tools/webpack-jumpstart', '--partial'], check=True)
+                run([f'{self.name}/tools/webpack-jumpstart', '--partial'], check=False)
```

Comment:
I know that this is the default (unfortunately), but I want to make it explicit that this is ok to fail.